src/topp/AssayGeneratorMetabo.cpp

```diff
@@ -23,9 +23,13 @@
 #include <OpenMS/KERNEL/RangeUtils.h>
 #include <OpenMS/KERNEL/StandardTypes.h>
 #include <OpenMS/SYSTEM/File.h>
-#include <QDir>
+#include <QtCore/QDir>
+#include <QtCore/QDirIterator>
+#include <QtCore/QString>
+#include <QtCore/QProcess>
 #include <algorithm>
 #include <map>
+#include <boost/algorithm/string.hpp>
```

Comment:
needed?

Reply:
Not needed any more. Parsing the compound.info file with CsvFile.